plugins/cosmic_reach_model_editor/cosmic_reach_model_editor.js

```diff
@@ -0,0 +1,287 @@
+
+function sumArrays(a, b){
+    s = []
+    for(let i = 0; i < Math.min(a.length, b.length); i++){
+        s.push(a[i] + b[i])
+    }
+    return s
+}
+
+(() => {
+    let codec, format, export_action, import_action, dialog, properties
```

Comment:
`sumArray` should be defined in the plugin's local scope and not exposed globally.
```suggestion
(() => {
    function sumArrays(a, b){
        s = []
        for(let i = 0; i < Math.min(a.length, b.length); i++){
            s.push(a[i] + b[i])
        }
        return s
    }
    let codec, format, export_action, import_action, dialog, properties
```